automated/linux/kselftest/kselftest.sh

```diff
@@ -6,7 +6,8 @@
 OUTPUT="$(pwd)/output"
 RESULT_FILE="${OUTPUT}/result.txt"
 LOGFILE="${OUTPUT}/kselftest.txt"
-TESTPROG="kselftest_armhf.tar.gz"
+#TESTPROG="kselftest_armhf.tar.gz"
+TESTPROG="kselftest_aarch64.tar.gz"
```

Comment:
@zzdxxd
you can overwrite this variable using -t option for the script. Is there a reason aarch64 should be the default? 

@nareshkamboju 
maybe we should detect the architecture and use it in determining the file name?